website/docs/topics/the-anatomy-of-unleash.mdx

```diff
@@ -8,17 +8,17 @@ This guide's purpose is to give you a conceptual overview of how Unleash works.
 The end of this guide presents a [short use case, explaining how you might configure Unleash](#use-case) to start working with feature toggles.
 
 
-## The global level
+## The root level
```

Comment:
Ok, back to my other comment, maybe because of this concept of "root level" makes sense to also change to "root resources"